bool primary_button 
bool secondary_button 
bool grip_button
bool trigger_button
float64 trigger_value